test/Microsoft.Azure.SignalR.Management.Tests/HttpRequestFacts.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Azure.SignalR.Management.Tests
+{
+    public class HttpRequestFacts
+    {
+        [Fact]
+        public async Task RequestAndResponse()
```

Comment:
>RequestAndResponse [](start = 26, length = 18)

Fact naming convention: `{Action}{State}{ExpectedResult}`.
Also, for UT we use mock httpclient instead of rely on network. This case fails when the computer is offline.

Reply:
Remove this test, since it doesn't test code in SDK.
Thanks for comments for naming.